sylt-common/src/op.rs

```diff
@@ -235,13 +235,21 @@ pub enum Op {
     /// Does not affect the stack.
     Define(usize),
     /// A helper instruction for the typechecker,
-    /// *assumes* top value on the stack
+    /// *Assumes* top value on the stack
```

Comment:
Either a small `a` in `assumes` and the comma above, or a capital `A` in `Assumes` and a period above.